internal/services/validation_service.go

```
@@ -672,6 +696,9 @@ func (s *ValidationService) persistAuditEventWithTx(ctx context.Context, tx pgdb
 		LimitUsageDetails: resp.LimitUsageDetails,
 	}
 
+	// Extract metricsFactory for observability on error path
+	_, _, _, metricsFactory := libCommons.NewTrackingFromContext(ctx)
```

golangci-lint (2.11.4)

[error] declaration has 3 blank identifiers

(dogsled)